lib/core/versions/latest/FeeManager.ts

```diff
@@ -17,41 +17,54 @@ export default class FeeManager {
    *
    * @throws if the number of operations are <= 0.
    */
-  public static convertNormalizedFeeToTransactionFee (normalizedFee: number, numberOfOperations: number, feeMarkupFactor: number): number {
+  public static convertNormalizedFeeToTransactionFee (normalizedFee: number, numberOfOperations: number, feeMarkupPercentage: number): number {
 
     if (numberOfOperations <= 0) {
       throw new SidetreeError(ErrorCode.OperationCountLessThanZero, `Fee cannot be calculated for the given number of operations: ${numberOfOperations}`);
     }
 
-    const normalizedFeePerOperation = normalizedFee * ProtocolParameters.normalizedToPerOperationFeeFactor;
-    const perOperationFee = normalizedFeePerOperation * numberOfOperations;
+    const normalizedFeePerOperation = normalizedFee * ProtocolParameters.normalizedFeeToPerOperationFeeMultiplier;
+    const normalizedFeeForAllOperations = normalizedFeePerOperation * numberOfOperations;

```

Comment:
Nit & optional: consider just `feePerOperation` and `feeForAllOperations` as I don't see these fee being "normalized".